examples/example4.go

```diff
@@ -57,12 +60,17 @@ func main() {
 			defer output.Release()
 			signal = input.Signal()
 		} else {
+			// If this is not the first input file, the number of Channels
+			// and the Rate must match that of the first file (WHY?)
```

Comment:
Just a note, this is coming from sox - it doesn't do any format conversion unless told to do so. So for example mixing mono and stereo requires inserting either a channel split or a channel merge filter...